src/anshuKJun20/SumOfIntegerByAscii.java

```diff
@@ -0,0 +1,20 @@
+package anshuKJun20;
+
+public class SumOfIntegerByAscii {
+
+	int getSumOfAllNumnersInJava(String input) {
+		int sum = 0;
+		for (int index = 0; index < input.length(); index++) {
+			if (input.charAt(index) >= '0' && input.charAt(index) <= '9') {
+				sum += (input.charAt(index) - '0');
```

Comment:
Nice! Without using valueOf,Interger.parseInt methods  you calculated sum.

Reply:
sum = sum + Character.getNumericValue(str.charAt(index);